src/core/storage/Pager.cpp

```diff
@@ -1,6 +1,8 @@
 #include <array>
 #include <fstream>
 #include <vector>
+#include <algorithm>
+#include <filesystem>
```

Comment:
Algorithm and filesystem imports are unused

Reply:
:+1: